src/Modules/ModulesGraded.jl

```diff
@@ -1426,29 +1426,32 @@ of `F` in form of a Betti table.
 Alternatively, use `betti`.
 
 # Examples
-```julia
+```jldoctest
 julia> R, (w, x, y, z) = graded_polynomial_ring(QQ, [:w, :x, :y, :z]);
 
 julia> I = ideal(R, [x*z, y*z, x*w^2, y*w^2])
-ideal(x*z, y*z, w^2*x, w^2*y)
+Ideal generated by
+  x*z
+  y*z
+  w^2*x
+  w^2*y
 
 julia> A, _= quo(R, I)
-(Quotient of multivariate polynomial ring by ideal with 4 generators, Map from
-R to A defined by a julia-function with inverse)
+(Quotient of multivariate polynomial ring by ideal (x*z, y*z, w^2*x, w^2*y), Map: R -> A)
 
 julia> FA  = free_resolution(A)
 Free resolution of A
 R^1 <---- R^4 <---- R^4 <---- R^1 <---- 0
 0         1         2         3         4
 
 julia> betti_table(FA)
-       0  1  2  3
-------------------
-0    : 1  -  -  -
-1    : -  2  1  -
-2    : -  2  3  1
-------------------
-total: 1  4  4  1
+     |  0  1  2  3
+-----+------------
+0    |  1  -  -  -
+1    |  -  2  1  -
+2    |  -  2  3  1
+-----+------------
+total:  1  4  4  1
```

Comment:
this looks a bit off-centered. I suggest to remove one horizontal space between the `|` and the following entry, i.e. like this
```suggestion
     | 0  1  2  3
-----+-----------
0    | 1  -  -  -
1    | -  2  1  -
2    | -  2  3  1
-----+-----------
total: 1  4  4  1
```

Reply:
I changed it because to me it seemed weird that all entries had two space to their left, except those in the left most column *shrug*.

Reply:
Interesting to hear this pov. In my mind it was *obvious* that each column has the form space-elem-space
Dunno, I don't have any preference here